docs/src/components/docs-card/DocsPageLayout.vue

```diff
@@ -219,6 +219,7 @@ $primary-color: #38b2ac;
   cursor: pointer;
   padding: 8px 0px;
   font-weight: 400;
+  list-style-type: none;
```

Comment:
It will be nice if we can add this as an global style

Reply:
I have made the changes @hrushi-thorat

Reply:
Let me know if you meant something else